src/github/pull-request.ts

```diff
@@ -60,6 +76,6 @@ export default async (context: Context, jiraClient, util): Promise<void> => {
 		return;
 	}
 
-	context.log(`Sending pullrequest update to Jira ${jiraClient.baseURL}`)
+	context.log(`Sending pullrequest update to Jira ${jiraClient.baseURL}`);
```

Comment:
I'd like to add the pull request ID here, so we can search for a combination of Jira URL and PR id